README.md

```diff
@@ -498,6 +498,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 
   * [Appinvento](https://appinvento.io/) — AppInvento is a free No code app builder. In the automatically generated backend code, users have complete access to the source code and unlimited APIs and routes, allowing for extensive integration. The free plan includes three projects, five tables, and a Google add-on.
   * [Cody AI](https://sourcegraph.com/cody) - Cody is a coding AI assistant that uses AI and a deep understanding of your codebase to help you write and understand code faster. Cody gives developers a choice of LLMs (including local inference), has support for various IDEs, supports all popular programming languages, and has a free plan. The free plan gives developers 20 chat messages (using Claude 3 Sonnet as the LLM) and 500 autocompletions (using the Starcoder 16b) each month.
+  * [Continue](https://github.com/continuedev/continue) - Continue keeps developers in flow. Our open-source VS Code and JetBrains extensions enable you to easily create your own modular AI software development system that you can improve
```

Comment:
Please detail what SaaS features this have that are free?

Reply:
Everything is open-source, so they are all free, but I can add something explicit if you don't feel that is obvious enough

Reply:
This list is for SaaS offerings. 

So open source fine but that’s a side issue. Without a SaaS component it does not fit.

Reply:
Fixed to address your requirements for this list

Reply:
Sorry it really just isn’t suitable for the list.

Reply:
Thanks for the submission but we’ll pass on this one.

Reply:
Could you please clarify why it doesn't? We have a free tier. Please read the latest version

You don't even have to log in. You click "Install" once in [VS Code](https://marketplace.visualstudio.com/items?itemName=Continue.continue) or [JetBrains](https://plugins.jetbrains.com/plugin/22707-continue) marketplace, and it just works

Reply:
You provide a free trial not free tier
Very few messages 
You don’t provide the execution of modules users have to bring their own keys for that. 

It just doesn’t fit the list sorry. Please read the top of the readme.

Reply:
Sorry today is not my day. I now see the checkboxes I somehow overlooked twice (!!) and see how we don't meet your definition. Thank you for consideration.

I think it's rather unfortunate that you are not inclusive of an offering that is truly free (i.e. open source), with a free trial that is more generous than months of an equivalent a free tier SaaS already listed, and that can be used without having to login unlike those SaaS equivalents, but it is your list, so I respect your opinion

Reply:
It’s not about inclusive it’s about the purpose of the list. 

You are blaming a bunch of grapes for not being inclusive to apples. 

There are 100s of awesome lists several with focussed on a topic that would suit your product. This list just doesn’t overlap with your offering.

Reply:
As I understand the README, the purpose of this list is to help infrastructure developers (System Administrator, DevOps Practitioners, etc.) find offerings that are likely to be useful to them and have a free developer tier, so they can make informed decisions without wasting a lot of time. This what I understand to be a grape.

I was dumb and didn't initially follow the instructions + then later used the words "free trial" instead of "free tier", causing you to view and treat us as an apple, when we are more of a grape than others that were approved. We are on the grey line, and I missed my chance to explain how we fit on the inclusive side of that line. Thank you for your time; it's made reconsider how we might benefit from the words "SaaS" and "free tier" :)